docs/contribute/05_reference.qmd

```diff
@@ -33,5 +33,17 @@ class TestConf(BackendTest):
         ...
 ```
 
-{{< include ../reference/BackendTest.qmd >}}
-{{< include ../reference/ServiceBackendTest.qmd >}}
+```{python}
+#| echo: false
+#| output: asis
+import os
+
+paths = [
+    "../reference/BackendTest.qmd",
+    "../reference/ServiceBackendTest.qmd",
+]
+
+for path in filter(os.path.exists, paths):
+    with open(path) as f:
+        print(f.read())
```

Comment:
I tried using

```python
    print(f"{{{{< include {path} >}}}}")
```

but that shows this warning

```
WARNING (/nix/store/8879djd960kr6waqa41a4ypxd0sc34fk-quarto-1.4.339/share/filters/main.lua:13666) Shortcode 'include' not found
```

And doesn't render anything.